<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBankAccount</class>
 <widget class="QWidget" name="AddBankAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Bank Account</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2c3e50;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    color: #ecf0f1;
}

QGroupBox {
    border: 2px solid #3498db;
    border-radius: 10px;
    margin-top: 10px;
    color: white;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 3px;
    background-color: #2c3e50;
}

QLabel {
    color: white;
}

QPushButton#logoutButton {
    background-color: #e74c3c;
    color: white;
    font-size: 14px;
    padding: 8px 16px;
    border: none;
    border-radius: 8px;
}


QLineEdit {
    background-color: #ecf0f1;
    border: 1px solid #bdc3c7;
    border-radius: 6px;
    padding: 5px;
	color:rgb(0,0,0);
}

QPushButton {
    background-color: #27ae60;
    color: white;
    padding: 8px 12px;
    font-weight: bold;
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #2ecc71;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <widget class="QPushButton" name="logoutButton">
     <property name="text">
      <string>Back To Dashboard</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="AddAccount_Layout">
        <item>
         <widget class="QLabel" name="LL_2">
          <property name="styleSheet">
           <string notr="true">QLabel {
    font-size: 15px;
    font-weight: bold;
    color: rgb(0, 255, 234);
    padding: 10px 0;
    border-bottom: 2px solid #3498db;
    letter-spacing: 1px;
}
</string>
          </property>
          <property name="text">
           <string>Add Bank Account</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="customerName_Label">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Account Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Please select...</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Savings</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Checking</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Qarz_al_Hasan</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Card Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="CN_LineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Generate automatically</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>IBAN</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="IBAN_LineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Generate automatically</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Account Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AN_LineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Generate automatically</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>CCV2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Generate automatically</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>PIN</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="PIN_LineEdit">
            <property name="maxLength">
             <number>4</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Static Second Password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="SSP_LineEdit">
              <property name="maxLength">
               <number>5</number>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::EchoMode::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QDateEdit" name="dateEdit">
           <property name="maximumDate">
            <date>
             <year>2035</year>
             <month>12</month>
             <day>31</day>
            </date>
           </property>
           <property name="minimumDate">
            <date>
             <year>2025</year>
             <month>5</month>
             <day>14</day>
            </date>
           </property>
           <property name="displayFormat">
            <string>yy/MM</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="currentSectionIndex">
            <number>0</number>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Initial Balance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Balance_LineEdit"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="Add_btn">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="Search_Layout">
        <item>
         <widget class="QLabel" name="LL">
          <property name="styleSheet">
           <string notr="true">QLabel {
    font-size: 15px;
    font-weight: bold;
    color: rgb(0, 255, 234);
    padding: 10px 0;
    border-bottom: 2px solid #3498db;
    letter-spacing: 1px;
}
</string>
          </property>
          <property name="text">
           <string>Add Bank Account</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>National ID Number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="getID_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="getID_btn">
            <property name="text">
             <string>Search Customer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
